src/checkers/inference/InferenceMain.java

```diff
@@ -420,15 +429,15 @@ public static boolean isHackMode(boolean condition) {
             if (traces[2].getMethodName().equals("isHackMode")) {
                 hackLocation = traces[3];
             }
-            getInstance().logger.warning("Encountered hack: " + hackLocation);
+            getInstance().logger.info("Encountered hack: " + hackLocation);
```

Comment:
Log files are filled up with "encountered hack" messages, and it's not surprising that inference encountered hack: if we don't enable hack feature, CFI mostly doesn't work on every project. So I make the logging importance down to info to make viewing log easy and let real warnings noticable.